app/Http/Controllers/BlogController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Controllers;
```

Comment:
Do not forget to follow up on these automatic generated comments. During the exam, points will be deducted when we find those in your code